pages/dkp/konvoy/2.2/dkp-upgrade/index.md

```diff
@@ -9,6 +9,18 @@ beta: false
 
 The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the most recent improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped, or on-prem environment to the latest version of DKP.
 
+
+## Upgrade paths
+
+Verified upgrade paths are as follows:
```

Comment:
Since this is 2.x, I think maybe we just need the 2s in here.

Reply:
Which I realize don't have any yesses yet unless Kevin tells us differently.

Reply:
Yea, I hope we can support 2.1 > 2.2!